R/survey_summary_US_West_Coast.R

```diff
@@ -0,0 +1,55 @@
+# Explorations of US West Coast trawl surveys
+
+# surveys to consider
+survey_names <- c("Triennial", "AFSC.Slope", "NWFSC.Slope", "NWFSC.Combo")
+# extract data and bind into single dataframe
+catch <- NULL
+for (survey in survey_names) {
+  catch <- rbind(catch, 
+    nwfscSurvey::pull_catch(
+      common_name = "Pacific spiny dogfish", # random species
+      survey = survey
+    )
+  )
+}
+
+# use shorter names for surveys
+catch <- catch |> dplyr::mutate(
+    survey_name = dplyr::case_when(
+      Project == "Groundfish Triennial Shelf Survey" ~ "Triennial",
+      Project == "AFSC/RACE Slope Survey" ~ "AFSC Slope",
+      Project == "Groundfish Slope Survey" ~ "NWFSC Slope",
+      Project == "Groundfish Slope and Shelf Combination Survey" ~ "WCGBTS"
+    ) |> as.factor()
+  )
+
+# transform some dates
```

Comment:
We should either add an offset to the years for surveys with overlapping years or use different pch options by survey for visibility.